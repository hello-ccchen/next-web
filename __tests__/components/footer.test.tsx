/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import Footer from "@/components/footer";

it("renders Footer component unchanged", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
