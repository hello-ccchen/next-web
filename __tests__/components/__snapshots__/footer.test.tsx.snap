// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component unchanged 1`] = `
<div>
  <footer
    class="fixed-bottom bg-light p-3"
  >
    <span
      class="small text-muted"
    >
      Made by THREEC Â©
    </span>
  </footer>
</div>
`;
